test/functional/apps/visualize/_histogram_request_start.js

```diff
@@ -30,7 +30,8 @@ export default function ({ getService, getPageObjects }) {
         log.debug(`Interval = ${providedInterval}`);
         await PageObjects.visualize.setNumericInterval(providedInterval);
         await PageObjects.visualize.clickGo();
-        await PageObjects.header.waitUntilLoadingHasFinished();
+        await PageObjects.common.sleep(1000);
```

Comment:
Should we add the sleep to the clickGo() method?
Or add a call to `waitForVisualization()` in the clickGo method?
https://github.com/liza-mae/kibana/blob/3e3f76d09b6eb916249c1c833b2c7d3613dfa0db/test/functional/page_objects/visualize_page.js#L824

Reply:
I believe I tried both those for this function and the only that helped was a sleep.